rust/src/composepost.rs

```diff
@@ -172,18 +172,19 @@ fn postprocess_presets(rootfs_dfd: &openat::Dir) -> Result<()> {
     Ok(())
 }
 
-// We keep hitting issues with the ostree-remount preset not being
-// enabled; let's just do this rather than trying to propagate the
-// preset everywhere.
+/// Ensure our own `%_dbpath` macro exists in the tree.
 fn postprocess_rpm_macro(rootfs_dfd: &openat::Dir) -> Result<()> {
     let rpm_macros_dir = "usr/lib/rpm/macros.d";
     rootfs_dfd.ensure_dir_all(rpm_macros_dir, 0o755)?;
     let rpm_macros_dfd = rootfs_dfd.sub_dir(rpm_macros_dir)?;
-    rpm_macros_dfd.write_file_with("macros.rpm-ostree", 0o644, |w| -> Result<()> {
+    let macro_filename = "macros.rpm-ostree";
```

Comment:
Were you originally thinking to factor this out into a separate function or a `const` or something?  What's the value of hoisting it to a variable as is?

Reply:
Ah right, without realizing I got attracted by the `rpm_macros_dir` just above. Good catch, they can both be consts.